code/modules/research/experimentor/experimentor.dm

```diff
@@ -0,0 +1,322 @@
+#define SCANTYPE_POKE 1
+#define SCANTYPE_IRRADIATE 2
+#define SCANTYPE_GAS 3
+#define SCANTYPE_HEAT 4
+#define SCANTYPE_COLD 5
+#define SCANTYPE_OBLITERATE 6
+#define SCANTYPE_DISCOVER 7
+#define SCANTYPE_RECYCLE 8
+
+#define RARITY_COMMON 0
+#define RARITY_UNCOMMON 1
+#define RARITY_RARE 2
+#define RARITY_VERYRARE 3
+
+/////////////////////////EXPERI-MENTOR DEFINITION/////////////////////////////
+//
+//
+//
+
+/obj/machinery/r_n_d/experimentor
+	name = "E.X.P.E.R.I-MENTOR"
+	icon = 'icons/obj/machines/heavy_lathe.dmi'
+	desc = "A marvel of technology, the onboard AI is capable of turning compatible material into brand-new contraptions... sometimes."
+	icon_state = "h_lathe"
+	var/list/allowedobjects = list(/obj/item/discovered_tech, /obj/item/unknown_tech)
+	density = 1
+	anchored = 1
+	use_power = IDLE_POWER_USE
+	var/base_flex_cost = 10
+	var/precise_scanner = 0
+	var/spareparts = 0
+	var/datum/experimentor/loot_definer/LootDefiner = new/datum/experimentor/loot_definer()
```

Comment:
can just use new() here. It will default to the specified type on the left hand side.